components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/open-sample-file-dialog.js

```diff
@@ -128,66 +124,54 @@ define(['require', 'lodash','jquery', 'log', 'backbone', 'file_browser', 'worksp
                 var openSampleConfigModal = sampleFileOpen.filter("#openSampleConfigModal");
                 var openFileWizardError = sampleFileOpen.find("#openFileWizardError");
                 var location = sampleFileOpen.find("input").filter("#location");
+                var locationSearch= sampleFileOpen.find("input").filter("#locationSearch");
 
-                var treeContainer  = sampleFileOpen.find("div").filter("#fileTree");
-                fileBrowser = new FileBrowser({container: treeContainer, application:app, fetchFiles:true,
-                showWorkspace:false,showSamples:true});
-
-                fileBrowser.render();
-                this._fileBrowser = fileBrowser;
-
-                //Gets the selected location from tree and sets the value as location
-                this.listenTo(fileBrowser, 'selected', function (selectedLocation) {
-                    var pathAttributes = selectedLocation.split(self.pathSeparator);
-                    var fileName = _.last(pathAttributes);
-                    var sampleOpenButton = sampleFileOpen.find("button").filter("#openButton");
-                    self._artifactName = pathAttributes[pathAttributes.length - 2];
-
-                    if(selectedLocation && fileName.lastIndexOf(".siddhi") != -1){
-                        location.val(fileName);
-                        sampleOpenButton.prop({
-                            disabled: false
-                        });
-                    } else {
-                        location.val('');
-                        sampleOpenButton.prop({
-                            disabled: true
-                        });
-                    }
-                });
-
-                sampleFileOpen.find("button").filter("#openButton").click(function () {
+                var treeContainer  = sampleFileOpen.find("div").filter("#sampleTable");
+                var bodyUlSampleContent = $('<ul class="recent-files clearfix"></ul>');
+                bodyUlSampleContent.attr('id', "sampleList");
+                var browserStorage = app.browserStorage;
+                var workspaceServiceURL = app.config.services.workspace.endpoint;
+                var getSampleDescServiceURL = workspaceServiceURL + "/listFiles/samples/descriptions";
+                var samplesWithDes={};
```

Comment:
Could you please format this JS code? there are several places without formatting

Reply:
Sure. Fixed in f228ff475e8dd19e1ef997ba24528c91a61ff51e